tests/application_development/code_relocation/src/test_file3.c

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2012-2014 Wind River Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/zephyr.h>
+#include <zephyr/sys/printk.h>
+#include <ztest.h>
+
+uint32_t var_file3_sram_data = 10U;
+uint32_t var_file3_sram2_bss;
+
+ZTEST(code_relocation, test_function_in_split_multiple)
```

Comment:
how this could possibly fail? You are not checking anything in here.

Reply:
when  relocate fails the exection will fail, and they will be exception.